### ISSUE TITLE
Select without default

### ISSUE THREAD

**Issue by mozfet** [mozfet/meteor-autoform-materialize#26]
v3.5.1 - When creating a select without a default or initial value, the input is rendered showing the first item in the list as being selected. This does not actually change the value of the input, but misleads the user into thinking something is selected, when it really is not.

**Comment by mozfet** [comment]
This will be fixed together with #23.

**Comment by mozfet** [comment]
This cannot be reproduced in playground using example, this works normally:
```
new SimpleSchema({
  selectWithoutDefault: {
    type: String,
    allowedValues: ['VALUE1', 'VALUE2'],
    autoform: {
      type: 'select'
    }
  },

  selectWithoutDefaultWithPlacehorder: {
    type: String,
    allowedValues: ['VALUE1', 'VALUE2'],
    autoform: {
      type: 'select',
      placeholder: 'Holding a place...'
    }
  },

  selectWithDefault: {
    type: String,
    allowedValues: ['VALUE1', 'VALUE2'],
    autoform: {
      type: 'select',
      default: 'VALUE1'
    }
  }
}, { tracker: Tracker});
```

![screen shot 2018-02-23 at 12 37 45](https://user-images.githubusercontent.com/5113240/36590155-60017e8c-1896-11e8-92a4-923c4d45eab4.png)
